### ISSUE TITLE
能否量化所有的操作，比如适量/适时大概之类的词能否用具体数字描述，已经准备好了秒表/量筒/天平等测量仪器

### ISSUE THREAD

**Issue by ivanzhuyufeng** [Anduin2017/HowToCook#74]

**Comment by Howard20181** [comment]
~~用仪器测量出来而做出的菜没有灵魂~~ 

平时做饭都是凭感觉的，比如米饭放水的量都是拿手指量的

**Comment by Daoctor** [comment]
没错 我也以为会量化呢，结果感觉就是个噱头. 番茄鸡蛋 也不说番茄多重 鸡蛋多重，只说数量. 至少提供一个标准 就可以自己适当增减了. 还有别的， 就是粘贴菜谱而已， 根本不严谨. 人的口味可以适应一定的误差 但有个准确的参考才有意义.

**Comment by recolic** [comment]
初中的时候就知道，米饭放水的量要用同一个碗来计算`米/水比例`，而不是拿手指量。
拿手指量最后是一定要挨骂的）

**Comment by kongdayan** [comment]
的确，小时候一根手指头刚好，现在半根就差不多了。
> 初中的时候就知道，米饭放水的量要用同一个碗来计算`米/水比例`，而不是拿手指量。 拿手指量最后是一定要挨骂的）

**Comment by willshion** [comment]
比例 -模糊-单元测试/试味

**Comment by cairenbin** [comment]
菜谱现在都很难量化的，1勺，是调料罐的小勺，还是饭店大厨的大炒勺。。。

中国菜的口味其实完全看个人喜好，按个大致的配料量烧几次，再按自己口味改进就行了，北方人的比例不一定适合南方人，南方人的也一样不适合北方人
